PlasterTemplate/PlasterTemplates/GitLab/PlasterManifest.xml

```diff
@@ -16,12 +16,17 @@
     <parameter name="GitLabUserName" type="text" prompt="Module author's GitLab user account name" />
     <parameter name="GitLabEmail" type="text" prompt="Module author's GitLab email address" />
     <parameter name="GitLabURL" type="text" prompt="The URL of the GitLab server" />
+    <parameter name="GitLabHostname" type="text" prompt="The Hostname or IP of the GitLab server, without http(s):// or /" />
     <parameter name="ModuleName" type="text" prompt="Name of your module" />
     <parameter name="ModuleDesc" type="text" prompt="Brief description on this module" />
     <parameter name='PSRepository' type='choice' default='0' store='text' prompt='Do you want to deploy to a custom repository or the PSGallery?'>
       <choice label='&amp;Custom repository' help="Deploy to a custom repository" value="CustomRepo"/>
       <choice label='&amp;PSGallery' help="Deploy to the public PSGallery" value="PSGallery"/>
     </parameter>
+    <parameter name='DeployDocs' type='choice' default='0' store='text' prompt='Do you want to deploy docs with mkDocs?'>
+      <choice label='&amp;Yes' help="Deploy docs with mkDocs" value="Y"/>
+      <choice label='&amp;No' help="Do not deploy docs with mkDocs" value="N"/>
+    </parameter>
```

Comment:
Can you move this to underneath the `PSRepositoryURL` parameter to keep the PSRepository questions together and then change the values to the full words to fit stylistically with previous choices?

Reply:
Oh and change default to 1 please as you mentioned in chat 🙂

Reply:
Could you also add a line in the readme just explaining that this requires setting up a PAT in order for the CI/CD tool to have write permissions to the repository to update the docs?